test/functional/wallet_createwallet.py

```diff
@@ -25,6 +25,7 @@ def add_options(self, parser):
 
     def set_test_params(self):
         self.num_nodes = 1
+        self.extra_args = [["-deprecatedrpc=walletwarningfield"]]
```

Comment:
Why do you set this flag for the entire test? Since the `warnings` is the new thing going forward, wouldn't it make more sense to write all the tests with only that field by default? And just add one test case with the flag to check `warning` was also included?